src/Microsoft.ML/Runtime/EntryPoints/InMemoryDataView.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+
+[assembly: LoadableClass(typeof(void), typeof(InMemoryDataView), null, typeof(SignatureEntryPointModule), "InMemoryDataView")]
+namespace Microsoft.ML.Runtime.EntryPoints
+{
+    public class InMemoryDataView
```

Comment:
>InMemoryDataView [](start = 17, length = 16)

Where is this being used? I don't see any references to this class in code or tests.

Reply:
This is a entrypoint :) Data.DataViewReference it get used in MemoryCollection.cs (At least it entry point wrapper)

---
In reply to: [187222065](https://github.com/dotnet/machinelearning/pull/106#discussion_r187222065) [](ancestors = 187222065)

Reply:
Ok, silly me as looking for "InMemoryDataView" not "Data.DataViewReferenc"

---
In reply to: [187222365](https://github.com/dotnet/machinelearning/pull/106#discussion_r187222365) [](ancestors = 187222365,187222065)